src/core/utils/helpers/url.ts

```diff
@@ -82,9 +82,10 @@ export const constructMaterialUrl = (
   workId: WorkId,
   type?: string
 ) => {
-  const materialUrl = url;
+  const materialUrl = new URL(url.href);
```

Comment:
It is a minor thing, and it is more or less the same. But I think you should clone the URL object like we have done in the other [url helper functions](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/50cf62e4ff18d8f1bd771b0a1693f84bcd314532/src/core/utils/helpers/url.ts)

So instead of:
```typescript
const materialUrl = new URL(url.href);
```
You write: 
```typescript
const materialUrl = new URL(url);
```
The stringify method of `URL`makes sure that the string representation of `url` is sent to the new constructor.